--- lib/python3.10/site-packages/notebook/services/contents/filecheckpoints.py	2021-04-14 17:39:54.621051563 +0100
+++ filecheckpoints.py	2021-04-14 17:42:06.405989920 +0100
@@ -29,7 +29,7 @@
     """
 
     checkpoint_dir = Unicode(
-        '.ipynb_checkpoints',
+        os.path.join(os.environ["SNAP_USER_COMMON"], ".ipynb_checkpoints"),
         config=True,
         help="""The directory name in which to keep file checkpoints
 
