--- filecheckpoints.py	2023-09-02 16:17:34.818494056 +0100
+++ root/lib/python3.12/site-packages/jupyter_server/services/contents/filecheckpoints.py	2023-09-02 16:32:03.357168874 +0100
@@ -30,7 +30,7 @@
     """
 
     checkpoint_dir = Unicode(
-        ".ipynb_checkpoints",
+        "/tmp/jupyroot checkpoints",
         config=True,
         help="""The directory name in which to keep file checkpoints
 
