--- serverapp.py	2023-09-02 16:17:16.826146476 +0100
+++ root/lib/python3.10/site-packages/jupyter_server/serverapp.py	2023-09-02 16:01:59.539904529 +0100
@@ -1260,7 +1260,7 @@
             return not addr.is_loopback
 
     use_redirect_file = Bool(
-        True,
+        False,
         config=True,
         help="""Disable launching browser by redirect file
      For versions of notebook > 5.7.2, a security feature measure was added that
