--- root/build/unix/compiledata.sh	2021-04-14 15:58:10.500738972 +0100
+++ compiledata.sh	2021-04-14 16:57:30.540341620 +0100
@@ -142,4 +142,7 @@
    mv ${COMPILEDATA}.tmp $COMPILEDATA; fi
 )
 
+sed -i "s#-isystem${SNAPCRAFT_PART_INSTALL}/usr/include/x86_64-linux-gnu##g" $COMPILEDATA
+sed -i "s#-isystem${SNAPCRAFT_PART_INSTALL}/usr/include##g" $COMPILEDATA
+sed -i "s#-isystem${SNAPCRAFT_STAGE}/include##g" $COMPILEDATA
 exit 0
