--- root-patched/gui/webdisplay/src/RWebDisplayHandle.cxx       2025-07-16 08:40:19.564454440 +0100
+++ root/gui/webdisplay/src/RWebDisplayHandle.cxx       2025-07-16 10:01:03.843386066 +0100
@@ -243,19 +243,7 @@
 FILE *RWebDisplayHandle::BrowserCreator::TemporaryFile(TString &name, int use_home_dir, const char *suffix)
 {
    std::string dirname;
-   if (use_home_dir > 0) {
-      if (use_home_dir == 1) {
-         const char *tmp_dir = gSystem->TempDirectory();
-         if (tmp_dir && (strncmp(tmp_dir, "/home", 5) == 0))
-            use_home_dir = 0;
-         else if (!tmp_dir || (strncmp(tmp_dir, "/tmp", 4) == 0))
-            use_home_dir = 2;
-      }
-
-      if (use_home_dir > 1)
-         dirname = gSystem->GetHomeDirectory();
-   }
-   return gSystem->TempFileName(name, use_home_dir > 1 ? dirname.c_str() : nullptr, suffix);
+   return gSystem->TempFileName(name, std::getenv("SNAP_REAL_HOME"), suffix);
 }

 static void DummyTimeOutHandler(int /* Sig */) {}
